packages/rule-https-only/README.md

```diff
@@ -0,0 +1,83 @@
+# https-only (`https-only`)
+
+`https-only` checks if your site is using HTTPS and warns against having
+mixed content.
+
+## Why is this important?
+
+HTTPS is important to guarantee content integrity. Even when your site
+doesn't have sensitive information, an attacker can change the content
+or inject malicious scripts (like a [crypto miner][crypto miner] to
+use your users CPU power).
```

Comment:
We should also add a note that some browser features are not restricted to HTTPS only.

See also: https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts/features_restricted_to_secure_contexts